quiz.js

```diff
@@ -0,0 +1,68 @@
+var arr=[];
+var index = 0;
+//store user input controls & submit button
+var submitBtn = document.getElementById("submit");
+var height = document.getElementById("height");
+var character = document.getElementById("character");
+
+
+
+function growTree(obj){
+//modified the function to take an object as the argument
+	obj.height = document.getElementById("height").value;
+	obj.character = document.getElementById("character").value;
+
```

Comment:
See if you can prevent the rest of the code from running if the user has not entered anything in the input fields.